src/Elliptic/DiscontinuousGalerkin/Penalty.hpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+
+namespace elliptic::dg {
+
+/*!
```

Comment:
I can't find this in the documentation as it seems the dg namespace within elliptic is undocumented

Reply:
Pushed a commit that adds the namespace to the docs